feature/experimental/policy/policy_vrf_selection/otg_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -43,7 +43,7 @@ const (
 	vlan20            = 20
 	ipipProtocol      = 4
 	ipv6ipProtocol    = 41
-	ipv4Address       = "198.18.0.1/32"
+	ipv4Address       = "192.0.2.2/32"
```

Comment:
This is in the ateSrc IP address space. Update to `192.0.2.13`?

Reply:
ok..  I changed the flow srcIp to point to "198.18.0.1" instead